### PR TITLE
Add icons to structure view

### PR DESCRIPTION
**Pull request by YuriyYatsyk**

### CODE REVIEW

File: src/main/java/com/appian/intellij/k/KStructureViewElement.java [hunk 1]
```diff
@@ -116,7 +115,10 @@ public String getLocationString() {
       @Nullable
       @Override
       public Icon getIcon(boolean unused) {
-        return element.isInternal() ? AllIcons.Nodes.C_private : AllIcons.Nodes.C_public;
+        if (fnDefinition.isPresent())
```
Comment (a2ndrade) [inline src/main/java/com/appian/intellij/k/KStructureViewElement.java]:
```suggestion
        if (fnDefinition.isPresent()) {
```

File: src/main/java/com/appian/intellij/k/KStructureViewElement.java [hunk 2]
```diff
@@ -116,7 +115,10 @@ public String getLocationString() {
...
+          return element.isInternal() ? KIcons.PRIVATE_FUNCTION : KIcons.PUBLIC_FUNCTION;
+        else
```
Comment (a2ndrade) [inline src/main/java/com/appian/intellij/k/KStructureViewElement.java]:
```suggestion
        } else {
```

File: src/main/java/com/appian/intellij/k/KStructureViewElement.java [hunk 3]
```diff
@@ -116,7 +115,10 @@ public String getLocationString() {
...
+          return element.isInternal() ? KIcons.PRIVATE_VARIABLE: KIcons.PUBLIC_VARIABLE;
```
Comment (a2ndrade) [inline src/main/java/com/appian/intellij/k/KStructureViewElement.java:121]:
```suggestion
          return element.isInternal() ? KIcons.PRIVATE_VARIABLE: KIcons.PUBLIC_VARIABLE;
        }
```

**Comment by a2ndrade** [comment]
@YuriyYatsyk could you please:

1) rebase your branch to remove the merge commit?
2) update the conditional block to include braces (to match the code style of the rest of the codebase)

Thanks!

**Review by a2ndrade — CHANGES_REQUESTED** [review]
@YuriyYatsyk could you please:

1) rebase your branch to remove the merge commit?
2) update the conditional block to include braces (to match the code style of the rest of the codebase)

Thanks!

File: src/main/java/com/appian/intellij/k/KStructureViewElement.java (hunk 3 above)
Comment (YuriyYatsyk) [reply src/main/java/com/appian/intellij/k/KStructureViewElement.java:121]: done

**Comment by YuriyYatsyk** [comment]
Are there any other issues with pull request? If you'd rather not accept these changes, please say so. 
I have other improvements that I would like to contribute, but I don't want pull requests just hanging there. Is there some process that you'd like me to follow?

**Comment by a2ndrade** [comment]
@YuriyYatsyk I apologize for the delay. I'll definitely merge these changes (and any other improvements you may have) Thanks! I just hadn't had time to do it. I'll do it in the following days, though. Thank you for your patience.

**Pull request merged**